src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3150,6 +3150,18 @@ internal bool ShouldSuppressNullableAnnotations(Symbol definition)
             return true;
         }
 
+        private bool HasAllowedNullableAssemblyOptOut(AssemblySymbol assembly)
```

Comment:
>HasAllowedNullableAssemblyOptOut [](start = 21, length = 32)

The name feels somewhat confusing to me. Perhaps "HaveNullableOptOutFor" or something similar?